alchemist-loading/src/test/kotlin/it/unibo/alchemist/test/TestExportInterval.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2010-2019, Danilo Pianini and contributors listed in the main project's alchemist/build.gradle file.
+ *
+ * This file is part of Alchemist, and is distributed under the terms of the
+ * GNU General Public License, with a linking exception,
+ * as described in the file LICENSE in the Alchemist distribution's top directory.
+ */
+
+package it.unibo.alchemist.test
+
+import io.kotest.core.spec.style.StringSpec
+import it.unibo.alchemist.loader.InitializedEnvironment
+import it.unibo.alchemist.loader.LoadAlchemist
+import io.kotest.matchers.shouldNotBe
+import it.unibo.alchemist.loader.export.CSVExporter
+import it.unibo.alchemist.model.interfaces.Position
+import org.junit.jupiter.api.Assertions.assertNotNull
+import org.kaikikm.threadresloader.ResourceLoader
+
+class TestExportInterval<T, P : Position<P>> : StringSpec({
+    "test export interval" {
+        val file = ResourceLoader.getResource("testExportInterval.yml")
+        assertNotNull(file)
+        val loader = LoadAlchemist.from(file)
+        assertNotNull(loader)
+        val initialized: InitializedEnvironment<T, P> = loader.getDefault()
+        val exporter = initialized.exporters.firstOrNull {
+            it is CSVExporter
+        }
+        if (exporter is CSVExporter) {
+            exporter.interval shouldNotBe 1.0
+        }
```

Comment:
Also, fail if it's not the correct type. You can use a `require(exporter is CSVExporter)` and then do business as usual.

Reply:
Done.

Reply:
This is still very broken. Require runs the block in case of failure!